champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/LevelField.java

```diff
@@ -35,14 +36,8 @@ public String getName() {
     @Override
     public String[] getDescription(int level) {
         return new String[]{
-                "For every enemy within <stat>" + radius + "</stat> blocks,",
-                "You deal <val>" +  (10 + ((level - 1) * 5)) + ".0%</val> extra damage",
-                "",
-                "For every ally <stat>" + radius + "</stat> blocks,",
-                "You deal <val>" + (10 + ((level - 1) * 5)) + ".0%</val> less damage",
-                "",
-                "Maximum extra damage: <stat>60%",
-                "Minimum extra damage: <val>"+ (60 - ((level - 1) * 15)),
+                "For every enemy that you are outnumbered by within <stat>" + radius + "</stat> blocks,",
+                "you deal <val>" + (level * damagePerPlayer) + "</val> more and take <val>" + (level * damagePerPlayer) + "</val> less damage",
```

Comment:
Bro doesnt english

For every enemy that you are outnumbered by within x blocks, you:

Deal y more damage
Take z less damage

-- Any other information that might be relevant